test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,22 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest-" + string(uuid.NewUUID())
+		ns, err := f.CreateNamespace(namespaceName, nil)
+		namespaceName = ns.ObjectMeta.Name
```

Comment:
If an error happens at the above `CreateNamespace()` call, `ns` will be nil and this line 255 makes an exception.
It is better to check `err` like line 256 before the above line.

Reply:
Good observation, updated in 2754cc37ea5